packages/datadog-plugin-mongodb-core/test/core.spec.js

```diff
@@ -115,9 +115,11 @@ describe('Plugin', () => {
             agent
               .use(traces => {
                 const span = traces[0][0]
-                const resource = `find test.${collection} {"_id":"?"}`
+                const resource = `find test.${collection}`
```

Comment:
This would change how customers query data in their dashboards, right? Would this normally be a patch update?

Reply:
Nevermind, `queryInResourceName` lets the user opt-in.